### DISCUSSION TITLE
'Reset my password' button on 'Reset password' page should send you back to the tenant login page after click.

### DISCUSSION THREAD

**Discussion by cfredericksen** [fief-dev/fief#392] [Ideas]
My users get confused when they click 'Reset my password' and nothing occurs. They dont know where to go after they click 'Reset my password'. They end of clicking 'Reset my password'  again and get an error. They dont realize their password was changed and they have to login again.

**Comment by frankie567** [comment]
Hmm, I see. This redirection happens, but only if the user is currently involved in a login session (i.e. triggered by an OAuth2 flow from your app):

https://github.com/fief-dev/fief/blob/9c7e72e4d37c078c722a5445bc12a6c1e159b274/fief/apps/auth/routers/reset.py#L96-L100

We could very well redirect even if we don't have that session, but bear in mind that, in this context, users will be redirected to the Fief's [user dashboard](https://docs.fief.dev/user-journey/#user-dashboard) after login.

**Comment by cfredericksen** [comment]
Thanks for the info. I think it would be a great option to add and redirecting to the user dashboard would force the user to the login, right? 

Here is my user workflow for context:
1. Administrator creates a new user, my app hits Fief's API and sets an arbitrary password that is embedded and not known.
2. New user receives the Welcome Email from Fief and step 1 is to "reset your password", link to {tenant}/forgot
3. New user sets new password (where the redirect to the user dashboard, to force a login, could take place)
4. New user attempts to Login to my app
5. New user is presented with verify email account
6. Once verify email account is complete, new user is presented with my app.

**Comment by frankie567** [comment]
Adding this as an issue: https://github.com/fief-dev/fief/issues/400
